SW_Flow_lib.c

```diff
@@ -1214,7 +1213,7 @@ void lyrSoil_to_lyrTemp_temperature(unsigned int nlyrSoil, double depth_Soil[],
   #ifdef SWDEBUG
   int debug = 0;
   #endif
-	double depth_Soil2[nlyrSoil + 1], sTemp2[nlyrSoil + 1];
+	double depth_Soil2[OVER_SIZE] = {0}, sTemp2[OVER_SIZE] = {0};

```

Comment:
Why not use existing constant for this specific purpose of soil layer number, i.e., `MAX_LAYERS`, if we want here a constant array size?